hazelcast/src/test/java/com/hazelcast/nio/serialization/compatibility/BinaryCompatibilityFileGenerator.java

```diff
@@ -82,6 +84,7 @@ private static SerializationService createSerializationService(ByteOrder byteOrd
             config.addSerializerConfig(serializerConfig);
         }
         config.setByteOrder(byteOrder);
+        config.getCompactSerializationConfig().addSerializer(new MainDTOSerializer());
```

Comment:
main dto also contains the inner dto. why don't we add a serializer for it?

Reply:
There's also NamedDTO. Looks like I missed them. Adding.